model/version/db.go

```diff
@@ -40,6 +40,7 @@ var (
 	IdentifierKey          = bsonutil.MustHaveTag(Version{}, "Identifier")
 	RemoteKey              = bsonutil.MustHaveTag(Version{}, "Remote")
 	RemoteURLKey           = bsonutil.MustHaveTag(Version{}, "RemotePath")
+	ActivatedKey           = bsonutil.MustHaveTag(Version{}, "Activated")
```

Comment:
Make this private if it's not being used anywhere else.

Reply:
As discussed, this should be deleted since this is no longer populated